platform/ios/app/MBXViewController.m

```diff
@@ -709,6 +724,79 @@ - (void)removeParkLayer
     [self.mapView.style removeLayer:parkLayer];
 }
 
+- (void)styleFilteredFill
+{
+    // set style and focus on Texas
+    [self.mapView setStyleURL:[NSURL URLWithString:@"mapbox://styles/justin/citc7ya7w00002jpa0z16cjah"]];
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(31, -100) zoomLevel:3 animated:NO];
+
+    // after slight delay, fill in Texas
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(2 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^
```

Comment:
A hard-coded timeout would be unfortunate. We don't have a completion block–based way to set the style; in the meantime, you can do this in didFinishLoadingMap or the forthcoming didFinishLoadingStyle perhaps.